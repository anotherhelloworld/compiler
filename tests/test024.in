var
var1, double1: double;