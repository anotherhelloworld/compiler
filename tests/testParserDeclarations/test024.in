var
    a: record
        b: integer;
        c: double;
    end;