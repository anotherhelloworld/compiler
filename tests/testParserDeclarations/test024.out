Var   a
   Record
      Var   b
         Type   integer

      Var   c
         Type   double

   End

